### ISSUE TITLE
toJavaScript() failed with "Cannot find type definition file for 'node'." on TypeScript@2.4.1

### ISSUE THREAD

**Issue by gfx** [teppeis/typescript-simple#82]
via espower-typescript:

```
Error: Cannot find type definition file for 'node'.
    at TypeScriptSimple.toJavaScript ($project/node_modules/typescript-simple/index.js:168:23)
    at TypeScriptSimple.compile ($project/node_modules/typescript-simple/index.js:69:25)
    at Object.loadTypeScript ($projectnode_modules/espower-typescript/index.js:19:22)
    at Module.load (module.js:503:32)
```

**Comment by teppeis** [comment]
It seems one of implicit breaking changes between typescript v2.4.0 and v2.4.1.
If you remove `compilerOptions.types` of `tsconfig.json`, this error is resolved.
But there is another issue https://github.com/power-assert-js/espower-typescript/issues/30 ...

**Comment by gfx** [comment]
Right, causing another issues:

```
$project/node_modules/typescript-simple/index.js:182
            var text = file.text;
                           ^
TypeError: Cannot read property 'text' of undefined
    at TypeScriptSimple.toJavaScript ($project/node_modules/typescript-simple/index.js:182:28)
```


I'll watch the TypeScript issue for now

**Comment by teppeis** [comment]
@gfx Fixed! Use espower-typescript@8.0.2
